### ISSUE TITLE
Differences in the new solver

### ISSUE THREAD

**Issue by K20shores** [NCAR/music-box#271]
@SancyW pointed out differences in the result of her configuraitons between the micm-powered music box and the camp-powered music box

Below are some screenshots and configurations showing difference. New is on the left, old on the right

# C1
configuration: [config_0522_C1.zip](https://github.com/user-attachments/files/17557434/config_0522_C1.zip)
<img width="959" alt="image" src="https://github.com/user-attachments/assets/c928e2da-7926-473e-ba91-21d9fcb8ebd5">

# C2
configuration: [config_0621_C2.zip](https://github.com/user-attachments/files/17557436/config_0621_C2.zip)
<img width="704" alt="image" src="https://github.com/user-attachments/assets/c1c22957-5cdf-4b8c-81dc-be847a7e08b7">
<img width="704" alt="image" src="https://github.com/user-attachments/assets/ddff4ad6-234f-447d-b69c-4d3b6390da1e">
<img width="736" alt="image" src="https://github.com/user-attachments/assets/a41bb594-2e05-4f41-81ad-d6947cf49894">
<img width="726" alt="image" src="https://github.com/user-attachments/assets/99e39699-f421-4960-af25-de0fa3f04343">
<img width="1049" alt="image" src="https://github.com/user-attachments/assets/8e367200-01e9-49be-b985-abad17a9f5be">

# C3
configuration: [config_0621_C3_perfect.zip](https://github.com/user-attachments/files/17557437/config_0621_C3_perfect.zip)
<img width="640" alt="PastedGraphic-1" src="https://github.com/user-attachments/assets/90363401-db2e-4599-9b9e-b7ebf4b07e7c">
<img width="640" alt="PastedGraphic-2" src="https://github.com/user-attachments/assets/4f8dbabb-79e6-4033-8a26-d5db83094415">

# C4
configuration: [config_0704_C4.zip](https://github.com/user-attachments/files/17557441/config_0704_C4.zip)
<img width="677" alt="image" src="https://github.com/user-attachments/assets/6b125f53-3aa5-410f-8ce3-e2076e4f4898">
<img width="697" alt="image" src="https://github.com/user-attachments/assets/48ee01ae-0f5a-496e-9e35-10bcc18ba810">
<img width="710" alt="image" src="https://github.com/user-attachments/assets/98874b77-570e-498d-937a-4390c7c6cd24">
<img width="719" alt="image" src="https://github.com/user-attachments/assets/21eb4a50-454c-4037-8c5a-77913e5c66b7">
<img width="951" alt="image" src="https://github.com/user-attachments/assets/cbdc2e5c-c298-4ccb-b940-d601fc3967af">

# C5
configuration: [config_C5_0730.zip](https://github.com/user-attachments/files/17557443/config_C5_0730.zip)
<img width="835" alt="image" src="https://github.com/user-attachments/assets/dc50b9bc-48f9-4ec9-9592-013bb0773f81">
<img width="858" alt="image" src="https://github.com/user-attachments/assets/f36bde9b-f5a5-47cc-97d4-905476170e11">
<img width="906" alt="image" src="https://github.com/user-attachments/assets/3cab7439-0311-4eb4-9fe8-64703b335c88">

**Comment by K20shores** [comment]
@SancyW, yes, that could be another difference. Adding an M would change the integrated reaction rate for that arrhenius reaction

**Comment by SancyW** [comment]
@K20shores That makes sense. A butterfly effect. Complicated non-linear impacts.

**Comment by SancyW** [comment]
![WX20241121-175649@2x](https://github.com/user-attachments/assets/da8a93f2-6764-473f-9403-187c84bab2aa)
The file failed to upload

**Comment by SancyW** [comment]
@K20shores Got it, that's good. BTW, could you send me the csv files of the old versions?

**Comment by K20shores** [comment]
> @K20shores Got it, that's good. BTW, could you send me the csv files of the old versions?

working on that now. I got tied up with other things things week. The only thing on my to do list right now is getting you these results

**Comment by SancyW** [comment]
@K20shores great, thank you so much!! I understand it. Take your time.

**Comment by SancyW** [comment]
Great work! Really grateful for your help!

**Comment by mattldawson** [comment]
May be similar to #308
